lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -65,6 +65,31 @@ def custom_build_settings
             settings
           end
 
+          # Filters the given resource file references discarding empty paths which are
```

Comment:
@benasher44 If you have better wording here, I'd like to hear it as I don't like it so much :)